modules/git/repo_tag.go

```diff
@@ -239,7 +238,7 @@ func (repo *Repository) GetTags() ([]string, error) {
 		return nil
 	})
 
-	version.Sort(tagNames)
+	// ToDo version.Sort(tagNames)
```

Comment:
do we realy have to Sort them?
otherwise they will come up the way thew are tagged - in my point of view this is more natural